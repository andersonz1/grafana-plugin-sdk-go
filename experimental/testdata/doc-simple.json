{
    "hello": 3,
    "string": "a",
    "values": [true, false, true],
    "mixed": ["a", true, 3],
    "sub": {
        "value": 2.34
    }
}