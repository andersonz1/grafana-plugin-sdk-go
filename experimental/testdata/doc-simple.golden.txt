ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] 
Name: 
Dimensions: 9 Fields by 1 Rows
+-----------------+-----------------+-----------------+-----------------+-----------------+-----------------+----------------+-----------------+-----------------+
| Name: hello     | Name: string    | Name: values[0] | Name: values[1] | Name: values[2] | Name: mixed[0]  | Name: mixed[1] | Name: mixed[2]  | Name: sub.value |
| Labels:         | Labels:         | Labels:         | Labels:         | Labels:         | Labels:         | Labels:        | Labels:         | Labels:         |
| Type: []float64 | Type: []*string | Type: []*bool   | Type: []*bool   | Type: []*bool   | Type: []*string | Type: []*bool  | Type: []float64 | Type: []float64 |
+-----------------+-----------------+-----------------+-----------------+-----------------+-----------------+----------------+-----------------+-----------------+
| 3               | a               | true            | false           | true            | a               | true           | 3               | 2.34            |
+-----------------+-----------------+-----------------+-----------------+-----------------+-----------------+----------------+-----------------+-----------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////QAQAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEDAAoADAAAAAgABAAKAAAACAAAAFAAAAACAAAAKAAAAAQAAABQ/P//CAAAAAwAAAAAAAAAAAAAAAUAAAByZWZJZAAAAHD8//8IAAAADAAAAAAAAAAAAAAABAAAAG5hbWUAAAAACQAAAEwDAADYAgAAZAIAAAACAACcAQAAOAEAANQAAABsAAAABAAAAOr8//8UAAAAQAAAAEAAAAAAAAADQAAAAAEAAAAEAAAA2Pz//wgAAAAUAAAACQAAAHN1Yi52YWx1ZQAAAAQAAABuYW1lAAAAAAAAAADS/P//AAACAAkAAABzdWIudmFsdWUAAABO/f//FAAAAEAAAABAAAAAAAAAA0AAAAABAAAABAAAADz9//8IAAAAFAAAAAgAAABtaXhlZFsyXQAAAAAEAAAAbmFtZQAAAAAAAAAANv3//wAAAgAIAAAAbWl4ZWRbMl0AAAAAIv7//xQAAABAAAAAQAAAAAAABgE8AAAAAQAAAAQAAACg/f//CAAAABQAAAAIAAAAbWl4ZWRbMV0AAAAABAAAAG5hbWUAAAAAAAAAABT+//8IAAAAbWl4ZWRbMV0AAAAAgv7//xQAAABAAAAAQAAAAAAABQE8AAAAAQAAAAQAAAAA/v//CAAAABQAAAAIAAAAbWl4ZWRbMF0AAAAABAAAAG5hbWUAAAAAAAAAAHT+//8IAAAAbWl4ZWRbMF0AAAAA4v7//xQAAABAAAAAQAAAAAAABgE8AAAAAQAAAAQAAABg/v//CAAAABQAAAAJAAAAdmFsdWVzWzJdAAAABAAAAG5hbWUAAAAAAAAAANT+//8JAAAAdmFsdWVzWzJdAAAAQv///xQAAABAAAAAQAAAAAAABgE8AAAAAQAAAAQAAADA/v//CAAAABQAAAAJAAAAdmFsdWVzWzFdAAAABAAAAG5hbWUAAAAAAAAAADT///8JAAAAdmFsdWVzWzFdAAAAov///xQAAABAAAAAQAAAAAAABgE8AAAAAQAAAAQAAAAg////CAAAABQAAAAJAAAAdmFsdWVzWzBdAAAABAAAAG5hbWUAAAAAAAAAAJT///8JAAAAdmFsdWVzWzBdABIAGAAUABMAEgAMAAAACAAEABIAAAAUAAAAPAAAAEAAAAAAAAUBPAAAAAEAAAAEAAAAkP///wgAAAAQAAAABgAAAHN0cmluZwAABAAAAG5hbWUAAAAAAAAAAAQABAAEAAAABgAAAHN0cmluZwAAAAASABgAFAAAABMADAAAAAgABAASAAAAFAAAAEQAAABMAAAAAAAAA0wAAAABAAAADAAAAAgADAAIAAQACAAAAAgAAAAQAAAABQAAAGhlbGxvAAAABAAAAG5hbWUAAAAAAAAAAAAABgAIAAYABgAAAAAAAgAFAAAAaGVsbG8AAAAAAAAA/////ygCAAAUAAAAAAAAAAwAFgAUABMADAAEAAwAAABYAAAAAAAAABQAAAAAAAADAwAKABgADAAIAAQACgAAABQAAABYAQAAAQAAAAAAAAAAAAAAFAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAgAAAAAAAAACAAAAAAAAAAAAAAAAAAAAAgAAAAAAAAACAAAAAAAAAAQAAAAAAAAAAgAAAAAAAAAGAAAAAAAAAAAAAAAAAAAABgAAAAAAAAACAAAAAAAAAAgAAAAAAAAAAAAAAAAAAAAIAAAAAAAAAAIAAAAAAAAACgAAAAAAAAAAAAAAAAAAAAoAAAAAAAAAAgAAAAAAAAAMAAAAAAAAAAAAAAAAAAAADAAAAAAAAAACAAAAAAAAAA4AAAAAAAAAAgAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAACAAAAAAAAABIAAAAAAAAAAAAAAAAAAAASAAAAAAAAAAIAAAAAAAAAFAAAAAAAAAAAAAAAAAAAABQAAAAAAAAAAgAAAAAAAAAAAAAAAkAAAABAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAABAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAABAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAIQAAAAAABAAAAYQAAAAAAAAABAAAAAAAAAAAAAAAAAAAAAQAAAAAAAAAAAAAAAQAAAGEAAAAAAAAAAQAAAAAAAAAAAAAAAAAIQLgehetRuAJAEAAAAAwAFAASAAwACAAEAAwAAAAQAAAALAAAADgAAAAAAAMAAQAAAFAEAAAAAAAAMAIAAAAAAABYAAAAAAAAAAAAAAAAAAAAAAAKAAwAAAAIAAQACgAAAAgAAABQAAAAAgAAACgAAAAEAAAAUPz//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAABw/P//CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAAkAAABMAwAA2AIAAGQCAAAAAgAAnAEAADgBAADUAAAAbAAAAAQAAADq/P//FAAAAEAAAABAAAAAAAAAA0AAAAABAAAABAAAANj8//8IAAAAFAAAAAkAAABzdWIudmFsdWUAAAAEAAAAbmFtZQAAAAAAAAAA0vz//wAAAgAJAAAAc3ViLnZhbHVlAAAATv3//xQAAABAAAAAQAAAAAAAAANAAAAAAQAAAAQAAAA8/f//CAAAABQAAAAIAAAAbWl4ZWRbMl0AAAAABAAAAG5hbWUAAAAAAAAAADb9//8AAAIACAAAAG1peGVkWzJdAAAAACL+//8UAAAAQAAAAEAAAAAAAAYBPAAAAAEAAAAEAAAAoP3//wgAAAAUAAAACAAAAG1peGVkWzFdAAAAAAQAAABuYW1lAAAAAAAAAAAU/v//CAAAAG1peGVkWzFdAAAAAIL+//8UAAAAQAAAAEAAAAAAAAUBPAAAAAEAAAAEAAAAAP7//wgAAAAUAAAACAAAAG1peGVkWzBdAAAAAAQAAABuYW1lAAAAAAAAAAB0/v//CAAAAG1peGVkWzBdAAAAAOL+//8UAAAAQAAAAEAAAAAAAAYBPAAAAAEAAAAEAAAAYP7//wgAAAAUAAAACQAAAHZhbHVlc1syXQAAAAQAAABuYW1lAAAAAAAAAADU/v//CQAAAHZhbHVlc1syXQAAAEL///8UAAAAQAAAAEAAAAAAAAYBPAAAAAEAAAAEAAAAwP7//wgAAAAUAAAACQAAAHZhbHVlc1sxXQAAAAQAAABuYW1lAAAAAAAAAAA0////CQAAAHZhbHVlc1sxXQAAAKL///8UAAAAQAAAAEAAAAAAAAYBPAAAAAEAAAAEAAAAIP///wgAAAAUAAAACQAAAHZhbHVlc1swXQAAAAQAAABuYW1lAAAAAAAAAACU////CQAAAHZhbHVlc1swXQASABgAFAATABIADAAAAAgABAASAAAAFAAAADwAAABAAAAAAAAFATwAAAABAAAABAAAAJD///8IAAAAEAAAAAYAAABzdHJpbmcAAAQAAABuYW1lAAAAAAAAAAAEAAQABAAAAAYAAABzdHJpbmcAAAAAEgAYABQAAAATAAwAAAAIAAQAEgAAABQAAABEAAAATAAAAAAAAANMAAAAAQAAAAwAAAAIAAwACAAEAAgAAAAIAAAAEAAAAAUAAABoZWxsbwAAAAQAAABuYW1lAAAAAAAAAAAAAAYACAAGAAYAAAAAAAIABQAAAGhlbGxvAAAAaAQAAEFSUk9XMQ==
